app/routes.rb

```diff
@@ -28,8 +28,12 @@ def route_slug_parts
     [params[:organization], params[:repository]].compact
   end
 
+  def protect!(role)
+    permissions.protect! role, slug.to_s
+  end
```

Comment:
This is already in mumukit service

https://github.com/mumuki/mumukit-service/blob/master/lib/mumukit/service/routes/auth.rb#L16